src/ToDoItem.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const ToDoItem = ({ item, toggleCompleteItem }) => {
```

Comment:
To be honest the word `toggleCompleteItem` took me a few seconds to understand what it's doing. Two verbs is actually quite confusing.

If you pay attention to most of the component design, you'll find that all their event handler starts with `onXXX`. For example `onClick`, `onScroll`, `onChange`. In your case you could name it `onComplete` or `onCompleteItem` and any other user will understand what the prop is doing clearly.